ideascube/serveradmin/catalog.py

```diff
@@ -322,6 +325,26 @@ def remove(self, install_dir):
         for path in glob(os.path.join(datadir, '*', zimname)):
             rm(path)
 
+    # [FIXME] Thoses two properties look like hacks.
+    # We may want to find a way to find those information in the package or
+    # catalog metadata.
+    # For now, use special cases.
+    @property
+    def theme(self):
+        base_name, _ = self.id.rsplit('.', 1)
+        if base_name in ("wikipedia", "wikivoyage", "vikidia"):
+            return "discover"
```

Comment:
> As a text. And it is translated in the template.

The issue is that if you don't call `gettext('xxx')` somewhere, with `xxx` hardcode, not a variable I mean, then `xxx` will never be discovered by gettext, then never added in .po files.